DEPENDENCIES

```diff
@@ -98,11 +98,31 @@ Prerequisites for building the OpenDataPlane (ODP) API
      --with-openssl-path=/home/${USER}/src/install-openssl-aarch64
    $ make
 
-3.3 Netmap packet I/O support (optional)
+3.3 ARMv8 Crypto native compile (optional)
+
+   Optimized crypto library using ARMv8 cryptographic extension for
+   AES-CBC, HMAC-SHA-1, HMAC-SHA-256 and AES-GCM algorithms.
+
+3.3.1 Building ARMv8 crypto library
+
+   # Checkout AArch64cryptolib code.
+   $ git clone https://github.com/ARM-software/AArch64cryptolib
```

Comment:
Let's just give all the necessary commands for a basic build:

`Checkout and build AArch64cryptolib code`
`...`
`$ cd AArch64cryptolib/`
`$ make`